app/src/main/java/com/github/se/icebreakrr/ui/profile/profilePictureEdit.kt

```diff
@@ -64,6 +65,13 @@ fun ImageCropperScreen(
           navigationActions.goBack()
         })
   }
+
+  BackHandler {
+    // for now makes impossible to quit the screen without cropping (as this may discard a
+    // valid temporary profile picture). A possible improvement for the future could be to add
+    // a slot to save the not cropped image so it doesn't discard the valid actual
+    // temporary image.
```

Comment:
This is a very good idea ! Maybe we should add it in the scrum board to make this feature available for the following weeks